template/build/webpack.prod.conf.js

```diff
@@ -98,7 +98,7 @@ var webpackConfig = merge(baseWebpackConfig, {
     ]),
     // service worker caching
     new SWPrecacheWebpackPlugin({
-      cacheId: 'my-vue-app',
+      cacheId: '{{ name }}',
```

Comment:
It may be worth ensuring we're applying convention (e.g namespacing by `-` so it uses `my-vue-app` instead of `My Vue App`) to input for the name here. Do we know if `{{name}}` has any sanitization done against it?